FriendlySkeletonWand/BasePlugin.cs

```diff
@@ -19,6 +19,7 @@
 using System.IO;
 using System.Linq;
 using UnityEngine;
+using static Mono.Security.X509.X520;
```

Comment:
Can you do me a favor and try removing all changes from this `BasePlugin.cs` file and test again? I suspect that all has been added here is some logging and it probably isn't needed. If everything still works, let's omit that.